tests/benchmarks/multicore/idle_pwm_loopback/sysbuild/nrf54h20dk_nrf54h20_cpurad.conf

```diff
@@ -0,0 +1 @@
+SB_CONFIG_REMOTE_BOARD="nrf54h20dk/nrf54h20/cpurad"
```

Comment:
Please remove and follow the remote board suggestion.
See also: https://github.com/nrfconnect/sdk-nrf/pull/19018#discussion_r1853707129

Reply:
removed